### DISCUSSION TITLE
Navigation par fenêtres glissantes

### DISCUSSION THREAD

**Discussion by mquandalle** [publicodes/publicodes#275] [Ideas]
La navigation proposée sur https://notes.andymatuschak.org est sympa je trouve, ça pourrait fonctionner pour la documentation.

Je l'ai découvert via [obsidian publish](https://obsidian.md/publish) qui a aussi des exemples avec un menu latéral, comme https://integralguide.com

**Comment by laem** [comment]
Intéressant ! En notant aussi que Wikipedia a expérimenté un tas de solutions, mais n'a jamais sauté le pas vers cette navigation. 
La popup sur bureau de prévisualisation d'une variable me semble déjà une super étape. De toutes façon pas dispo sur mobile, comme ta proposition plus complète, elles sont réservées au bureau. 
Wikipedia le fait et c'est super :)
